src/PHPImageWorkshop/Core/ImageWorkshopLayer.php

```diff
@@ -1528,8 +1528,11 @@ public function save($folder, $imageName, $createFolders = true, $backgroundColo
                     imagepng($image, $filename, $imageQuality);
                     unset($image);
                 }
+                return true;

```

Comment:
I think it be better to throw an exception if an error occured rather than return a boolean.